src/services/Libs.ts

```diff
@@ -238,6 +257,13 @@ export const localFileToRegex = (hostname: string) => {
   return hostname; // Doesn't have a file path...return as is.
 };
 
+/**
+ * Parse cookieStoreId for use in addExpressionUI...
+ */
+export const parseCookieStoreId = (contextualIdentities: boolean, cookieStoreId: string | undefined) => {
```

Comment:
Minor nit: I think this can be `cookieStoreId: string?`

Reply:
While I'd like to, that is actually an optional parameter per https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/Tab, not to mention Chrome doesn't have one.  The check within the function checks if it is defined otherwise assigns 'default' when returning for CAD lists.

Also when I tried removing it just now TypeScript is complaining, even though we'll always have the appropriate firefox permission to always have cookieStoreId as a parameter.